app/routes/clients.tsx

```diff
@@ -131,6 +183,22 @@ export default function Clients({ loaderData }: Route.ComponentProps) {
 
   return (
     <>
+      {/* Barre de recherche par email */}
+      <Form className="mb-4" onSubmit={handleSearch}>
+        <InputGroup>
+          <Form.Control
+            type="email"
+            placeholder="Rechercher par email exact"
+            value={emailSearch}
```

Comment:
No need to keep track of the email directly. Avoid setting the arguments programmatically in the `goToPage` method.